docker-compose-base.yml

```diff
@@ -23,8 +21,15 @@ server:
     - PYTHONDONTWRITEBYTECODE=1
     - DATABASE_URL=postgres://postgres@db/postgres
     - DEBUG=True
+    - INITIAL_ADMIN_USERNAME=admin
```

Comment:
Seems weird to hard-code config values here, but not in the EB config file.

Not sure how best to move to a single source of configuration truth, but would be nice to clean up in the future.